src/examples/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ set( contact_examples
      )
 
 
-set(example_depends axom tribol axom::cli11 axom::fmt)
+set(example_depends tribol)
 blt_list_append(TO example_depends ELEMENTS cuda IF ENABLE_CUDA )
```

Comment:
Is this one still needed?

Reply:
Unfortunately, the `cuda` dependency needs to be explicitly listed.